doc/build/dts/bindings-syntax.rst

```diff
@@ -19,6 +19,16 @@ like this:
 
 .. code-block:: yaml
 
+   # When the description text is too long, this field can
+   # be used to improve readability, e.g.:
+   #
+   # title: Binding the device's hardware model.
+   #
+   # description |
+   #   A piece of content with 20 lines.
+   #   ...
+   title: Concise title for the long description [optional]
```

Comment:
Thanks for the update. Please see the below text as well:

```
These keys are explained in the following sections.

.. _dt-bindings-description:

Description
***********
```

This is adding a key without adding a corresponding section; please fix that up too. Thanks!